packages/data-stores/src/reader/helpers/index.ts

```diff
@@ -88,20 +88,21 @@ const getSubscriptionMutationParams = (
 	action: 'new' | 'delete',
 	isLoggedIn: boolean,
 	blogId: number | string,
-	url?: string
+	url?: string,
+	emailId?: string
 ) => {
 	if ( isLoggedIn ) {
 		return {
 			path: `/read/following/mine/${ action }`,
 			apiVersion: '1.1',
-			body: { source: 'calypso', url: url },
+			body: { source: 'calypso', url: url, ...( emailId ? { email_id: emailId } : {} ) },
```

Comment:
Note to self: This is not used at the moment for the loggedIn endpoint. Still need to decide whether it is needed.